### ISSUE TITLE
https://news.walla.co.il - ads back  in chrome

### ISSUE THREAD

**Issue by MrTal** [AdguardTeam/AdguardFilters#27607]
Hi,
the ads here bac in chrome:
https://github.com/AdguardTeam/AdguardFilters/issues/26111
screenshot: http://prntscr.com/m0zffu
can you see here more info
uBlockOrigin/uAssets#4287 (comment)

**Comment by MrTal** [comment]
@AdamWr 
still only with adgurd i try with ublock and this work...

**Comment by AdamWr** [comment]
Hi,
Could you please check this rule:
`walla.co.il$$script[tag-content="JorLoad"][min-length="50000"][max-length="80000"]`
+
`walla.co.il#@#a[data-track-click]:not([href*="walla.co.il"]) img`
(second rule should unhide/unblock incorrectly blocked images)
or instead of first rule, this:
`walla.co.il#%#AG_abortInlineScript(/JorLoad/, 'decodeURIComponent');`

**Comment by BlazDT** [comment]
Adam, which Image is blocked wrongly? When I checked only ad ones were blocked.

**Comment by AdamWr** [comment]
Most images on https://news.walla.co.il/ are blocked/hidden on my end.
<details><summary>Screenshot</summary>

![image](https://user-images.githubusercontent.com/29142494/50551578-358a0e80-0c83-11e9-9e4c-71ff13299f19.png)

</details>

**Comment by MrTal** [comment]
@AdamWr 
@BlazDT 
yes, this work with filter this:
[walla.co.il$$script[tag-content="JorLoad"][min-length="50000"][max-length="80000"]
walla.co.il#@#a[data-track-click]:not([href*="walla.co.il"]) img](url)
many problems have with site this,
maybe Because the update filer latest in EasyListHebrew?
https://github.com/easylist/EasyListHebrew/commits/master
very strange, but now this work with filter this

**Comment by BlazDT** [comment]
Before removing that rule, please check for ads on main page or add another :not there Adam.
The rule was based on main page.

**Comment by MrTal** [comment]
@BlazDT 
1. still have popup:
https://healthy.walla.co.il/
https://sports.walla.co.il/
have popup too a few seconds and closed alone
Screenshot: http://prntscr.com/m1hnua

2. i  found anti adblock more:
in walla games cant see the game with adblock:
https://fun.walla.co.il/game/3205383
with adguard:
http://prntscr.com/m1hlzn
without adguard:
http://prntscr.com/m1hm8p

**Comment by AdamWr** [comment]
Could you please update filters and check if you still can reproduce these issues and ads?

**Comment by MrTal** [comment]
@AdamWr 
yes this work, i dont see now more ads

**Comment by MrTal** [comment]
@AdamWr 
@Alex-302 
the ads back again!
https://image.prntscr.com/image/UnB9KaiJTT6YfZxndmu3Cw.png
https://image.prntscr.com/image/RF6z6F-tR8ScEoj3WH-lUQ.png
https://image.prntscr.com/image/CstqhsHvQIe3lN3T9k_jog.png
http://prntscr.com/m3d2n9
https://news.walla.co.il/item/3209862

**Comment by AdamWr** [comment]
Could you please update filters and check if you still see ads?

**Comment by MrTal** [comment]
@AdamWr 
yes this work, but every week have new problem with and site this and the ads back to site this...
(can you see also here https://github.com/uBlockOrigin/uAssets/issues/4287#issuecomment-451656332)

**Comment by AdamWr** [comment]
>and there is a reason this filer: walla.co.il##+js(set-constant.js, String.fromCharCode, noopFunc
does not work work adguard?

AdGuard has a different syntax, this rule should works:
```
walla.co.il#%#AG_defineProperty('String.fromCharCode', { get: function() { return function() {}; }, set: function() {} });
```

**Comment by MrTal** [comment]
@AdamWr 
ok, i hope the ads no back again in the site this. tank you.
